### DISCUSSION TITLE
Remove pagination - display all product

### DISCUSSION THREAD

**Discussion by bamhost** [bootscore/bootscore#441] [Q&A]
Hello all,

Wordpress: 6.2
Bootscore Main: 5.2.3.2
Bootcommerce Child: I downloaded and installed from bootscore.me and it said it was 5.2.3.2 however, when I click on theme details in appearance it says the version is 5.2.0.0 (is this a typo in the template?)

Anyways, I am using the bootcommerce child template. I want to remove pagination for products and instead have all products show on one page instead of having to click to go to a next page. In Appearance > Customize > Woocommerce > Product Catalog I have the following settings:

Category Display: Show Products
Products Per Row: 4
Rows Per Page: 100 (this is plenty of rows to allow for all product to show on one page)

Those settings are not working. No matter what those settings are, the 4 products per row works, but rows per page does not. No matter what only 24 products are shown per page.

With this said, is there a settings inside bootscore main that is setting the actual pagination and/or the number of products per page? If yes, is there a way I can remove that limit?

Thank you for your help.

**Comment by justinkruit** [comment]
A function like this should do the trick:

```
function num_posts_archive_products($query) {
    if (!is_admin() && $query->is_archive('product') && $query->is_main_query()) {
            $query->set('posts_per_page', 100);
   }
    return $query;
}
add_action( 'pre_get_posts', 'num_posts_archive_products' );
```

**Reply by bamhost** [reply]
Justin...thank you so very very much. I pretty much figured I would need to add an action to the function file, but just needed to be sure. Again, thank you so very very much.

By the way, do you have any idea why the bootcommerce template version says 5.2.0.0 but the download says 5.2.3.2?

**Reply by crftwrk** [reply]
> By the way, do you have any idea why the bootcommerce template version says 5.2.0.0 but the download says 5.2.3.2?

This was a typo mistake in style.css. Fixed.

**Comment by crftwrk** [comment]
https://bootscore.me/documentation/functions/#Example_Pagination_amount_of_posts
